gabbi/runner.py

```diff
@@ -224,6 +227,14 @@ def _make_argparser():
         choices=['all', 'body', 'headers'],
         help='Turn on test verbosity for all tests run in this session.'
     )
+    parser.add_argument(
+        '--unsafe-yaml',
+        dest='safe_yaml',
+        action='store_false',
+        default=True,
+        help='Trun on recognization of Python object in additional of '
```

Comment:
Turn on recognition of Python objects in addition to standard YAML tags

Reply:
done